plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -4886,3 +4886,95 @@ func TestInterPodAffinityAnnotationsWithMultipleNodes(t *testing.T) {
 		}
 	}
 }
+
+func benchmarkPodMatchesNodeLabels(pods []*v1.Pod, nodes []*v1.Node, b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		for _, pod := range pods {
+			for _, node := range nodes {
+				podMatchesNodeLabels(pod, node)
+			}
+		}
+	}
+}
+
+func buildPodsAndNodes(podN, nodeN int) ([]*v1.Pod, []*v1.Node) {
+	hostNameLabel := "kubernetes.io/hostname"
+
+	hostNameGenFunc := func(i int) string {
+		return fmt.Sprintf("node_%d", i)
+	}
+	podNameGenFunc := func(i int) string {
+		return fmt.Sprintf("pod_%d", i)
+	}
+
+	pods := []*v1.Pod{}
+	nodes := []*v1.Node{}
+
+	for i := 0; i < podN; i++ {
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: podNameGenFunc(i),
+			},
+			Spec: v1.PodSpec{
+				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								{
+									MatchExpressions: []v1.NodeSelectorRequirement{
+										{
+											Key:      hostNameLabel,
+											Operator: v1.NodeSelectorOpIn,
+											Values:   []string{hostNameGenFunc(i % nodeN)},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		pods = append(pods, pod)
+
+	}
+
+	for i := 0; i < nodeN; i++ {
+		hn := hostNameGenFunc(i)
+		node := &v1.Node{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: hn,
+				Labels: map[string]string{
+					hostNameLabel: hn,
+				},
+			},
+		}
+		nodes = append(nodes, node)
+	}
+
+	return pods, nodes
+}
+
+func BenchmarkPodMatchesNodeLabels_500Pods_50Nodes(b *testing.B) {
```

Comment:
I found that other benchmark test function names not using `_`

s/BenchmarkPodMatchesNodeLabels_500Pods_50Nodes/BenchmarkPodMatchesNodeLabels500Pods50Nodes